### ISSUE TITLE
[game/items] BCSO Cake

### ISSUE THREAD

**Issue by ThatGuyJayXD** [InZidiuZ/op-framework-issue-tracker#1729]
**Assurance:**
Have you read through the rules from the `README.md` file in the root folder of this GitHub repository?
Yes

**Summary:**
So the idea i have is very simple and i know its not a priority in anyway just thought it was good/funny. So with the implementation of Sherrif Vedders Pigeon Milk i had an idea for yet another food item that would fit the meme well if you would. BCSO Cake aka Novak to anyone who knows PD well would be a food item like a piece of cake that would give the same amount of food as pigeon milk does hydration.

**Reason:**
So in short i dont think that this is groundbreaking or anything i just think it would be a good addition to have a food item that matches the hydration value of the pigeon milk equally.

TLDR
We need cake to go with our milk and balance out our diets. Item description - Novaks cake sure to make those criminals quake. or whatever you want :)

**Comment by InZidiuZ** [comment]
Closing as outdated. If this is still relevant, please create a new issue.
